symfony/messenger/6.0/post-install.txt

```diff
@@ -4,8 +4,10 @@
 
   * To send messages to a transport and handle them asynchronously:
 
-    1. Uncomment the <info>MESSENGER_TRANSPORT_DSN</> env var in <comment>.env</>
+    1. Update the <info>MESSENGER_TRANSPORT_DSN</> env var in <comment>.env</> if needed
        and <info>framework.messenger.transports.async</> in <comment>config/packages/messenger.yaml</>;
-    2. Route your message classes to the async transport in <comment>config/packages/messenger.yaml</>.
+    2. (if using doctrine) Generate a doctrine migration  <info>bin/console console doctrine:migration:diff</>
```

Comment:
some typos on this line:
`(if using Doctrine) Generate a Doctrine migration <info>bin/console doctrine:migration:diff</>`

Reply:
Thanks!